### DISCUSSION TITLE
Blank page, white page on mobile. Is there a software to see error on mobile?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/next.js#14857] [Help] [answered]
Hi my app run ok on desktop but on mobile it is blank, page loaded done but its just white page. Is there a software to run mobile browser to see what error like F12 on desktop?  Thank you.

**Comment by LucaNerlich** [comment]
On iOS: https://www.lifewire.com/activate-the-debug-console-in-safari-445798

Most likely, youll need some polyfills or need to call different methods.

**Reply by IRediTOTO** [reply]
this way is too hard.

**Comment by IRediTOTO** [comment, marked as answer]
@LucaNerlich  I found one, best for now browserstack.com. Hope this help full some one. I got one year free for this

**Reply by LucaNerlich** [reply]
Well, sure you can use browserstack - but thats a bit overkill. But if you get an extensive trial, go for it :)

**Comment by divyanshusahu** [comment]
You could also use [remote debugging](https://developers.google.com/web/tools/chrome-devtools/remote-debugging).

**Reply by IRediTOTO** [reply]
I dont have adroid, so hard to setup, but if you have an adroid this is the best way :D

**Comment by TSB-media-venture** [comment]
I deployed on netlify instead of vercel and it worked...
